beanstalk_worker/Dockerfile.workers

```diff
@@ -1,14 +1,17 @@
 FROM registry.centos.org/centos/centos
 MAINTAINER Bamacharan Kundu<bkundu@redhat.com>
+
 RUN yum update -y && \
     yum install python -y && \
+    yum install PyYAML -y && \
     yum clean all
 
 RUN mkdir -p /run_worker
-ADD beanstalkc.py ca.crt node.kubeconfig worker_start_build.py config.py /run_worker/
+
+ADD beanstalkc.py ca.crt node.kubeconfig worker_start_build.py \
```

Comment:
Yeah, your PR was created and merged while this PR was outstanding. I will update the Dockerfile.